feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -372,13 +373,19 @@ func (tc *testCase) testFlow(t *testing.T, packetSize uint16, ipHeader ondatra.H
 	octets := fpc.InOctets().Get(t) // Flow does not report out-octets.
 	outPkts := fpc.OutPkts().Get(t)
 	inPkts := fpc.InPkts().Get(t)
+
 	if outPkts == 0 {
 		t.Error("Flow did not send any packet")
 	} else if avg := octets / outPkts; avg > uint64(tc.mtu) {
 		t.Errorf("Flow source packet size average got %d, want <= %d (MTU)", avg, tc.mtu)
 	}
-	if p1InDiff.unicast < outPkts {
-		t.Errorf("DUT received too few source packets: got %d, want >= %d", p1InDiff.unicast, outPkts)
+
+	if pktlmtu {
+		t.Logf("Skipping packet check for packets larger than MTU")
```

Comment:
Why do we want to skip checking packets larger than MTU? If they are actually dropped, can we fix the counter?

Reply:
when packets are dropped they are not counted as unicast. if we want to do packet counter comparison for packet larger than  MTU,  I think we  should compare based on the dropped packet counters rather unicast packet.  we can explorer this as another pr if you want.